### PR TITLE
made CatF-is-Free more general

### PR DESCRIPTION
**Pull request by MirceaS**

### CODE REVIEW

File: src/Categories/Category/Construction/Graphs.agda [hunk 1]
```diff
@@ -529,12 +529,12 @@ CatF-is-Free o = record
     unwind-resp-≈ ε = Equiv.refl
     unwind-resp-≈ (x ◅ eq) = ∘-resp-≈ (unwind-resp-≈ eq) x
 
-  zig′ : (X : Graph o o o) → {A B : Graph.Obj X} → (f : Star (Graph._⇒_ X) A B) →
-    let Y = Free X in [ X ] (unwind Y) (mapGraph (GM X) f) ≈* f
```
Comment (MirceaS) [inline src/Categories/Category/Construction/Graphs.agda]: I have no idea why this originally was a `let ... in` construct. It seemed pointless to me so I refactored it but if I'm missing something let me know so I can change it back.

**Comment by sstucki** [comment]
LGTM, but I suggest we let @JacquesCarette have a look and do the merge.

**Comment by JacquesCarette** [comment]
This is definitely an improvement - thanks.

File: src/Categories/Category/Construction/Graphs.agda [hunk 2]
```diff
@@ -488,8 +488,8 @@ CatF = record
   open GraphMorphism≈
 
 -- Because of the Level changes in CatF, sizes must all be same:
```
Comment (MirceaS) [inline src/Categories/Category/Construction/Graphs.agda]: oops, forgot to remove the comment

Comment (JacquesCarette) [reply src/Categories/Category/Construction/Graphs.agda]: No worries, I'll take care of that later.

**Pull request merged**